onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -2234,10 +2234,10 @@ TEST(CApiTest, basic_cuda_graph) {
 #define cudaMemcpyHostToDevice hipMemcpyHostToDevice
 #define cudaMemcpyDeviceToHost hipMemcpyDeviceToHost
   Ort::MemoryInfo info_mem("Hip", OrtAllocatorType::OrtArenaAllocator, 0, OrtMemTypeDefault);
-#elif defined(USE_DML)
-  Ort::MemoryInfo info_mem("DML", OrtAllocatorType::OrtDeviceAllocator, 0, OrtMemTypeDefault);
-#else
+#elif defined(USE_CUDA) || defined(USE_TENSORRT)
   Ort::MemoryInfo info_mem("Cuda", OrtAllocatorType::OrtArenaAllocator, 0, OrtMemTypeDefault);
+#else
```

Comment:
```suggestion
#elif defined(USE_DML)
```
Consistent pattern with line 2220, and we shouldn't assume any *particular* EP is the default `else` case. I certainly agree though (along with Dmitri) with the intention of being able to test both CUDA and DML in the same test suite.

Reply:
Fair point, done.